debian/rules

```diff
@@ -8,6 +8,10 @@ export PYBUILD_NAME=eos-data-distribution
 
 override_dh_install:
 	dh_install --fail-missing
+	mkdir -p debian/edd-avahi-monitor/usr/bin
```

Comment:
the problem is that the python installer was putting these in the main package, `python-eos-data-distribution` any suggestions on how to fix that dh-python behaviour is welcome.